yarn.lock

```diff
@@ -5491,15 +5491,10 @@ csso@^4.0.2:
   dependencies:
     css-tree "^1.1.2"
 
-cssom@^0.4.4, "cssom@https://registry.npmjs.org/rrweb-cssom/-/rrweb-cssom-0.6.0.tgz":
+cssom@^0.4.4, cssom@^0.5.0, "cssom@https://registry.npmjs.org/rrweb-cssom/-/rrweb-cssom-0.6.0.tgz":
```

Comment:
should this file also be removed?

Reply:
you can just ignore this, there's two different tools that keep changing the format of this package in particular.

Reply:
ignore, as in keep it in the PR but it's a ignorable change?

Reply:
ah sorry, let's remove it from the PR